src/test/java/Homework21.java

```diff
@@ -0,0 +1,52 @@
+public class Homework21 extends LoginTests{
+
+
+    import org.openqa.selenium.By;
```

Comment:
In order to use a Java class declared in these packages, you need to include an import statement before class declaration in your program. The import statement starts with the keyword import, followed by the name of the java package.
==> You need to import library before `class Homework21`